Makefile

```diff
@@ -56,7 +56,8 @@ pipenv-dev-install: lib/Pipfile
 	# (Omitting this flag is causing incorrect dependency version
 	# resolution on CircleCI.)
 	cd lib; \
-		pipenv install --dev --skip-lock --sequential
+		rm Pipfile.lock; \
+		pipenv install --dev --sequential
```

Comment:
Can you add a comment here explaining that we're intentionally creating a lockfile for resolution determinism, even though we don't commit the lockfile to git?